source/api/cypress-api/selector-playground-api.md

```diff
@@ -37,6 +37,10 @@ Option | Accepts | Description
 - `attributes`
 - `nth-child`
 
+**{% fa fa-angle-right %} $el**  ***(Object)***
+
+The {% url 'JQuery element' http://api.jquery.com/Types/#jQuery %} that you want to get the selector value for.
+
```

Comment:
The `$el` argument for the `Cypress.SelectorPlayground.getSelector($el)` was missing so I documented it.